generator.yaml

```diff
@@ -55,6 +55,11 @@ resources:
           path: Events
       AuthToken:
         is_secret: true
+    hooks:
```

Comment:
Not sure but maybe you need a `sdk_find_post_set_output` hook as well? (similar to `dk_delete_pre_build_request.go.tpl`)

Reply:
It was not needed.
I have a upcoming separate PR to patch resource which is being deleted, so that kubectl describe provides latest state of the resource as it gets deleted.
But that is independent of this PR.